src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task{
+    protected String date;
+
+    public Event(String description, String date){
+        super(description);
+        this.date = date;
+    }
+    @Override
+    public String getDescription(){
+        return description + "(" + date + ")";
+    }
+    @Override
+    public String getType(){
+        return "E";
+    }
+}
```

Comment:
No violation of coding standard, PascalCase for Class and camelCase for variable and methods properly followed, good job!